package.json

```diff
@@ -60,6 +65,7 @@
     "babel-preset-react": "6.22.0",
     "bowser": "1.9.4",
     "cheerio": "1.0.0-rc.2",
+    "chromedriver": "75.1.0",
```

Comment:
@paulkaplan I remember us moving this out on purpose at some point in the past. Can you review?